pkgs/applications/misc/stylish/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
```

Comment:
Change the title of this commit:

`stylish: init at unstable-2022-12-05`

Reply:
Fixed.